app/views/admin/groups/_users.haml

```diff
@@ -1,17 +1,32 @@
+= simple_form_for :group_users, url: admin_group_users_path(@group) do |f|
+  .d-flex
+    .ml-auto.w-50
+      = f.input :user_ids, collection: User.where.not(id: @group.user_ids), as: :select_two, input_html: { multiple: true }, label: 'Users to Add'
+    .ml-4
+      = f.button :submit, class: 'btn btn-secondary mt-no-label' do
+        %i.icon-plus
+        Add Users
+
 - if @group.users.count.positive?
-  %h3= pluralize(@group.users.count, 'User')
+  %h3.mt-4= pluralize(@group.users.count, 'User')
   .card
     %table.table.table-striped.mb-0
       %thead
         %tr
           %th Name
           %th Email
+          %th
       %tbody
         - @group.users.each do |user|
           %tr
             %td= link_to_if can_edit_users?, user.name, edit_admin_user_path(user)
             %td= user.email
+            %td.d-flex
+              = link_to admin_group_user_path(@group, user), class: 'btn btn-sm btn-danger ml-auto', method: :delete, data: { confirm: "Are you sure you want to remove #{user.name} from #{@group.name}?" } do
```

Comment:
We should look into making these focusable for keyboard navigation

Reply:
That kind of optimization is always welcome.  Aren't links usually focusable?

Reply:
That's what I was expecting, too. Turns out it's an issue of coordinating settings between Firefox and OSX. This SO question led the way: https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox . So there's no issue with the code; just me on a new computer.